src/pages/instances/actions/CreateImageFromInstanceBtn.tsx

```diff
@@ -18,19 +25,28 @@ const CreateImageFromInstanceBtn: FC<Props> = ({ instance }) => {
     prohibitedStatuses.includes(instance?.status) ||
     Boolean(instanceLoading.getType(instance));
 
+  const handleOpen = (e: React.MouseEvent<HTMLButtonElement>) => {
+    openPortal(e);
+  };
+  const handleClose = () => {
+    closePortal();
+    onClose?.();
+  };
+
   return (
     <>
       {isOpen && (
         <Portal>
           <CreateImageFromInstanceForm
-            close={closePortal}
+            close={handleClose}
             instance={instance}
           />
         </Portal>
       )}
       <ActionButton
         appearance="default"
-        onClick={openPortal}
+        className={classNames("u-no-margin--bottom", classname)}
+        onClick={handleOpen}
```

Comment:
```suggestion
        onClick={openPortal}
```
The `handleOpen` function seems redundant. Same for two other buttons below.